### ISSUE TITLE
Upgrade from 2.0.0.7 to 2.0.0.8 - Benchmark required screen

### ISSUE THREAD

**Issue by billmcilhargey** [nicehash/NiceHashMiner2#122]
When upgrading Nicehash Miner 2 it looks like a benchmark is required (which is fine) but the benchmark screen pops under the initializing screen so you can't see the status or anything

![image](https://user-images.githubusercontent.com/19168174/27980573-13be693e-634e-11e7-97b9-6719787f4d44.png)

**Comment by fredhen** [comment]
I've had the same issue and was about to open an issue and saw this. I'll add to this:
Restarting and/or starting as administrator does not resolve the issue. I also closed the app and deleted the benchmark files to no avail.
So I proceeded to clicking the red bit sticking out and it was OK afterwards.
I haven't tested deleting the benchmarks again to try and replicate it. I will try that tomorrow.

EDIT: Closed the application, deleted the benchmark files and ran as administrator again and the screen did not get stuck on Loading Done screen. Perhaps only after an update?
